scripts/install_third_party_libs.py

```diff
@@ -106,11 +107,33 @@ def ensure_pip_library_is_installed(package, version, path):
             package, version, exact_lib_path)
 
 
+def ensure_system_python_libraries_are_installed(package, version):
+    """Installs the pip library with the corresponding version to the system
+    globally. This is necessary because the development application server
+    requires certain libraries on the host machine.
+
+    Args:
+        package: str. The package name.
+        version: str. The package version.
+    """
+    python_utils.PRINT(
+        'Checking if %s is installed.' % (package))
+    install_backend_python_libs.pip_install_to_system(package, version)
+
+
 def main():
     """Install third-party libraries for Oppia."""
     setup.main(args=[])
     setup_gae.main(args=[])
-    pip_dependencies = [
+    # These system python libraries are REQUIRED to start the development server
+    # and cannot be added to oppia_tools because the dev_appserver python script
+    # looks for them in the default system paths when it is run. Therefore, we
+    # must install these libraries to the developer's computer.
+    system_pip_dependencies = [
+        ('enum34', common.ENUM_VERSION),
+        ('protobuf', common.PROTOBUF_VERSION)
```

Comment:
Hi @kevjumba -- one question. Could you explain why we need this, again? I noticed that common.PROTOBUF_VERSION is 3.0.0 which is different from 3.13.0.

Reply:
I changed it back to 3.13.0. This is required to startup the dev_appserver(running it before the gae sandbox is created because dev_appserver is a script that searches for it in the system path).